frontend/scripts/build

```diff
@@ -26,3 +26,7 @@ rsync -avr resources/public/ target/dist/
 
 sed -i -re "s/\%version\%/$CURRENT_VERSION/g" ./target/dist/index.html;
 sed -i -re "s/\%buildDate\%/$BUILD_DATE/g" ./target/dist/index.html;
+
+# build storybook
+yarn run storybook:build || exit 1;
+rsync -avr storybook-static/ target/dist/storybook-static
```

Comment:
Not sure if `target/dist/storybook-static` is the right destination. Maybe `target/dist-storybook` so it's separate from the rest?